apps/camera/js/controllers/camera.js

```diff
@@ -73,7 +73,6 @@ CameraController.prototype.bindEvents = function() {
   app.on('settings:configured', this.onSettingsConfigured);
   app.on('previewgallery:opened', this.onGalleryOpened);
   app.on('previewgallery:closed', this.onGalleryClosed);
-  app.on('stoprecording', this.camera.stopRecording);
```

Comment:
the shutdownCamera call on hidden below does a stopRecording, so removing this seems like it should be fine.